### DISCUSSION TITLE
🛠️ Hacktoberfest 2023 - Guidelines

### DISCUSSION THREAD

**Discussion by qwerty541** [anuraghazra/github-readme-stats#3317] [Show and tell]
It's October, so that means that coders come out of the shadows worldwide to participate in this yearly's [https://hacktoberfest.com/](https://hacktoberfest.com/) 🚀. The guidelines for this amazing event can be found on [hacktoberfest.com](https://hacktoberfest.com/participation/). Since Github Readme Stats (GRS) this year also opted in as a participant in Hacktoberfest, here are some Github Readme Stats-specific guidelines. Feel free to comment below if anything is unclear.

Good Luck, and most of all, have fun! 🎉 

## What is Hacktoberfest

As explained on [the official site](https://hacktoberfest.com/):

> HACKTOBERFEST IS DIGITALOCEAN’S ANNUAL EVENT THAT ENCOURAGES PEOPLE TO CONTRIBUTE TO OPEN SOURCE THROUGHOUT OCTOBER. MUCH OF MODERN TECH INFRASTRUCTURE—INCLUDING SOME OF DIGITALOCEAN’S OWN PRODUCTS—RELIES ON OPEN-SOURCE PROJECTS BUILT AND MAINTAINED BY PASSIONATE PEOPLE WHO OFTEN DON’T HAVE THE STAFF OR BUDGETS TO DO MUCH MORE THAN KEEP THE PROJECT ALIVE. HACKTOBERFEST IS ALL ABOUT GIVING BACK TO THOSE PROJECTS, SHARPENING SKILLS, AND CELEBRATING ALL THINGS OPEN SOURCE, ESPECIALLY THE PEOPLE THAT MAKE OPEN SOURCE SO SPECIAL.
>
> FOR THE PAST 9 YEARS, THOUSANDS OF PEOPLE—CODERS AND NON-CODERS ALIKE—HAVE PARTICIPATED IN HACKTOBERFEST TO SUPPORT THE PROJECTS THEY USE AND LOVE, LEARN AND PRACTICE SKILLS THAT WILL ENHANCE THEIR CAREERS, AND MEET NEW PEOPLE WHO LOVE OPEN SOURCE AS MUCH AS THEY DO.

## How do I find issues I can work on for this year's Hacktoberfest

- The easiest way is to check issues with the [help-wanted](https://github.com/anuraghazra/github-readme-stats/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) label. These are the issues the maintainers have labelled as important. Within these issues, the issues with a [good-first-issue](https://github.com/anuraghazra/github-readme-stats/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) are best suited for new coders.
- Further, you can look at the [top-issues dashboard](https://github.com/anuraghazra/github-readme-stats/issues/1935) to see which issues are most upvoted.
- Lastly, you can create an issue yourself explaining what you want to approve to the GRS repository. We will review your issue and assign it the [hacktoberfest-accepted](https://github.com/anuraghazra/github-readme-stats/issues?q=is%3Aopen+is%3Aissue+label%3Ahacktoberfest-accepted) label if we think it is a good proposal that is doable within the Hacktoberfest time period.

## How do I get assigned to a Hacktoberfest issue

1. Comment on the issue you want to take on.
2. We will review your request, assign you to the issue and add a [hacktoberfest](https://github.com/anuraghazra/github-readme-stats/issues?q=is%3Aopen+is%3Aissue+label%3Ahacktoberfest) label to the issue.
3. Create a PR while making it clear that the PR is meant to be included in Hacktoberfest 2023.
4. We will review your PR and attach the [hacktoberfest-accepted](https://github.com/anuraghazra/github-readme-stats/issues?q=is%3Aopen+is%3Aissue+label%3Ahacktoberfest-accepted) label.
5. We will merge your PR into the master branch. 

## Which PR are excluded this year from GRS hacktoberfest

> ❗ PLEASE READ ❗.

- Due to the great amount of theme PR, [theme PRs](https://github.com/anuraghazra/github-readme-stats/pulls?q=is%3Aopen+is%3Apr+label%3Athemes) are excluded from this year's Hacktoberfest.
- Due to the difficulty of reviewing PR that contain languages which @anuraghazra do not speak all [translation-related PRs](https://github.com/anuraghazra/github-readme-stats/pulls?q=is%3Aopen+is%3Apr+label%3Adoc-translation) are excluded from this years Hacktoberfest.

**Comment by ebraj** [comment]
@qwerty541  I think there is a typo in this Guideline Readme, please check `How do I get assigned to a Hacktoberfest issue` - Step 3, Instead of 2022, it might be 2023, Is it?

**Reply by qwerty541** [reply]
Hey, @ebraj! Yes, you're right, it should be 2023. I corrected this mistake, thank you. I made this announcement based on last year's (https://github.com/anuraghazra/github-readme-stats/discussions/2076) and unfortunately missed it.

**Comment by qwerty541** [comment]
Closed since Hacktoberfest 2023 is over. Thanks, everybody, for your contributions! 🚀 See you next year. 👋🏻.
